submissions/MishaDerkach/html-css-popup/index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Popup</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <header class="header">
+        <nav class="nav">
+            <ul class="nav-list">
+                <li class="nav-list__item">
+                    <a href="#" class="nav-list__link">Mail</a>
+                </li>
+                <li class="nav-list__item">
+                    <a href="#" class="nav-list__link">Images</a>
+                </li>
+                <li class="nav-list__item popup">
+                    <input type="checkbox" id="popup-button" class="popup-input">
+                    <label for="popup-button" class="popup-button">
+                        <img src="popup-icons/popup-button.png" class="popup-icon">
+                    </label>
+
+                    <div class="content">
+                        <ul class="content-list">
+                            <li class="content-list__item">
+                                <a href="#" class="content-list__link">
+                                    <img src="popup-icons/apps.png" class="content-list__image">
+                                    <span class="content-list__title">Apps</span>
+                                </a>
+                            </li>
+                            <li class="content-list__item">
+                                <a href="#" class="content-list__link">
+                                    <img src="popup-icons/blogs.png" class="content-list__image">
+                                    <span class="content-list__title">Blogs</span>
+                                </a>
+                            </li>
+                            <li class="content-list__item">
+                                <a href="#" class="content-list__link">
+                                    <img src="popup-icons/calendar.png" class="content-list__image">
```

Comment:
```suggestion
    <img src="popup-icons/calendar.png" class="content-list__image" alt="">
```
`alt` attribute is required `img` attribute